embedding/esp8266/js/main.js

```diff
@@ -0,0 +1,4 @@
+function sysloop(ticknow) {
```

Comment:
Yes agree. It's same LED blink example and I want to make them work with one js source code. Maybe move them to some common folder and edit the Makefile for both. I'll work on this after this PR lands.